framework/src/scheduler/context.rs

```diff
@@ -105,22 +105,20 @@ impl NetBricksContext {
         }
     }
 
-    pub fn add_test_pipeline_to_core<
-        T: Fn(Vec<AlignedVirtualQueue>, &mut StandaloneScheduler) + Send + Sync + 'static,
-    >(
-        &mut self,
-        core: i32,
-        run: Arc<T>,
-    ) -> Result<()> {
+    /// Install a pipeline on a particular core.
```

Comment:
Are the changes in this file from `cargo fmt`?

Reply:
I just cleaned up the bounds here to keep the newer pattern :).